### ISSUE TITLE
expects import "github.com/kardianos/osext"

### ISSUE THREAD

**Issue by cslarson** [ethereum/go-ethereum#384]
the [Mist install script](https://github.com/ethereum/go-ethereum/wiki/Instructions-for-getting-the-Go-implementation-of-Ethereum-and-the-Mist-browser-installed-on-Ubuntu-14.04-(trusty)) now fails for me at this point:
```
cmd/utils/cmd.go:33:2: code in directory /home/carl/go/src/bitbucket.org/kardianos/osext expects import "github.com/kardianos/osext"
can't load package: cmd/utils/cmd.go:33:2: code in directory /home/carl/go/src/bitbucket.org/kardianos/osext expects import "github.com/kardianos/osext"
flags.go:33:2: code in directory /home/carl/go/src/bitbucket.org/kardianos/osext expects import "github.com/kardianos/osext"
```

**Comment by 5paceManSpiff** [comment]
I'm experiencing similar issues after running the following commands.
```
go get github.com/ethereum/go-ethereum/cmd/mist
go get github.com/ethereum/go-ethereum/cmd/ethereum
```

**Comment by tgerring** [comment]
Can you try `go get github.com/kardianos/osext`?

Then try to build to build Mist with:
`cd $GOPATH/src/github.com/ethereum/go-ethereum/cmd/mist && go install`

If all is well, you should then be able to `mist`

**Comment by tgerring** [comment]
A new [Mist install script](https://gist.github.com/tgerring/d4ab3f1672ed91a53c6c) has been linked to the wiki. Although the dependency error still exists when pulling the repo (indicated with comment on line 26), this is due to `go get` fetching master branch by default. Once #391 is merged into master, this problem should disappear.
